tests/profiler/core/test_profiler_config_parser.py

```diff
@@ -214,28 +171,24 @@ def test_dataloader_profiling_ranges(detailed_profiler_config_path, test_case):
 
 
 @pytest.mark.parametrize("test_case", python_profiling_test_cases)
-def test_python_profiling_ranges(python_profiler_config_path, test_case):
+def test_python_profiling_ranges(profiler_config_path, test_case):
```

Comment:
Add a note in the PR description, with the reason for these variable name changes.

Reply:
It's already in the description?

Reply:
> Use only one fixture for the profiling ranges tests since the profiler config is generated dynamically for those tests anyways

Reply:
Cool